### ISSUE TITLE
Define behavior for when a Category does not have elements

### ISSUE THREAD

**Issue by j-sal** [OpenRefine/CommonsExtension#43]
- Should a message be displayed?

**Comment by wetneb** [comment]
I would say:
* if a category exists, but does not contain any elements, then we should just create an empty project
* if a category does not exist, then we should display an error message in the form where the category name is input

**Comment by trnstlntk** [comment]
> I would say:
> 
> * if a category exists, but does not contain any elements, then we should just create an empty project
> * if a category does not exist, then we should display an error message in the form where the category name is input
> * if a category is a redirect to another category, we should probably follow that redirect and fetch the corresponding category

I agree with this 90%. Just the first bullet point: I can imagine that people will be confused / left in the dark with an empty project and no feedback at all, so if we can think of some message (however small) during the process, that would be great.

**Comment by trnstlntk** [comment]
For testing purposes, this is a list of redirected categories: https://commons.wikimedia.org/wiki/Special:WhatLinksHere?target=Template%3ACategory+redirect&namespace=14

One example: https://commons.wikimedia.org/wiki/Category:Professions

**Comment by trnstlntk** [comment]
Fetching category redirects on Commons is not straightforward. Redirects are produced by adding the {{Category redirect}} template to a category and it'd need some very dirty and Commons-specific code in OpenRefine to accommodate this scenario (either give the user some feedback about the redirect, or transparently following the redirect).

For now, this is a too complex task for the time we still have in the SDC project (less than 2 weeks) and we're not sure how necessary this will be (how frequently will users want to retrieve redirected categories and will be confused by empty results?). So we stall this issue 'for the future' for now.
